doc/devices/remote.rst

```diff
@@ -18,3 +37,90 @@ PennyLane device.
 	return backend
 
     dev = qml.device('qiskit.remote', wires=2, backend=configured_backend())
+
+.. warning::
+
+    Retrieving a backend from a provider has been deprecated and may not be supported. 
```

Comment:
Could we direct users towards an alternative in the case of expected deprecation?

Reply:
The alternative is the preferred use with `QiskitRuntimeService` (which is what the docs lead with). 

The reason I leave this is in is because certain other third-party backends still use this provider interface and I want to keep that available, but for IBM backends I think `QiskitRuntimeService` is the way to go. Maybe I should re-iterate that `QiskitRuntimeService` is what should be used.